models/dave_robot_models/description/rexrov/model.sdf

```diff
@@ -584,5 +584,22 @@
       <joint_name>thruster8_joint</joint_name>
     </plugin>
 
+    <!-- Sea Pressure Sensor Plugin -->
+    <plugin
+      filename="sea_pressure_sensor"
+      name=" dave_gz_sensor_plugins::SubseaPressureSensorPlugin">
+      <namespace>rexrov</namespace>
+      <!-- <link_name>base_link</link_name> -->
```

Comment:
I this `link_name` needed anytime later? if not should be removed

Reply:
same as above